test/fixtures/v3/program-orders.fixtures.js

```diff
@@ -3,8 +3,6 @@ const nock = require('nock');
 const RO = require('../../..');
 const { generateBasicAuthToken } = require('../../../lib/utils/auth');
 
-RO.config.set('apiVersion', 'v3');
```

Comment:
I suppose we should deprecate `v3` entirely, given we've done so in Pangea. Can you spool up a ticket for the modernization Epic, please?

Reply:
https://rewardops.atlassian.net/browse/MX-1213